lexicons/app/bsky/actor/getSuggestions.json

```diff
@@ -0,0 +1,44 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.actor.getSuggestions",
+  "type": "query",
+  "description": "Get a list of actors suggested for following. Used in discovery UIs.",
+  "parameters": {
+    "limit": {"type": "number", "maximum": 100},
+    "cursor": {"type": "string"}
+  },
+  "output": {
+    "encoding": "application/json",
+    "schema": {
+      "type": "object",
+      "required": ["actors"],
+      "properties": {
+        "cursor": {"type": "string"},
+        "actors": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": ["did", "handle", "actorType"],
+            "properties": {
+              "did": {"type": "string"},
+              "handle": {"type": "string"},
+              "actorType": {"type": "string"},
```

Comment:
I did the whole enum thing in the `getProfile` view: https://github.com/bluesky-social/atproto/blob/main/lexicons/app/bsky/actor/getProfile.json#L16

I guess it doesn't really matter because it's all strings in the end 😛 but wondering if we should decide on one or the other.

Would be a useful place for schema fragments

Reply:
Yeah that occurred to me as well. I decided we'll probably take another pass over schemas before long and to just do the simple thing for now.